test/regression/test_regression.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# This tests for backward compatibility to BASE_VERSION of sru:
```

Comment:
Is there any way we can test this in CCIE?

Reply:
forgot to reomte this file. removed in 1d3b1d7

Reply:
(so the regression test is now just a standard unit test, which is nice)

Reply:
(and it's super fast too..)